pom.xml

```diff
@@ -92,6 +92,68 @@
                 <module>tools</module>
             </modules>
        </profile>
+       <profile>
+            <id>no-buildver-default</id>
```

Comment:
should the above default profile just be combined into here?

Reply:
Maybe. It is not clear what logic guides the inclusion of modules like tools & api_validation. 

e.g.
301 contains  api_validation solely
311 contains tools and api_validation together

Reply:
ah right, actually we don't